### DISCUSSION TITLE
When someone is annoyed with you because you won't speak up, what might they say the cat has?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#313] [Galaxy Trivia] [answered]
**Hint:** 😝

**Comment by thuongtruong1009** [comment, marked as answer]
When someone is annoyed with you because you won't speak up, they might say "the cat has got your tongue." This is an idiom that means someone is unable to speak or is choosing not to speak, usually in a situation where they are expected to say something.
